package/translate/nl.po

```diff
@@ -160,15 +171,15 @@ msgstr "Hoogte:"
 
 #: ../contents/ui/config/ConfigAppearance.qml:149
 msgid "Spacing:"
-msgstr ""
+msgstr "Afstand:"
 
 #: ../contents/ui/config/ConfigAppearance.qml:157
 msgid "Fill opacity:"
-msgstr "Ondoorzichtigheid vullen:"
+msgstr "Opvulling van ondoorzichtigheid:"
```

Comment:
It's possible to have more shorten text to prevent layout issue ?

Reply:
I could shorten it to “Ondoorzichtigheid opvullen”. It's not as good, but still good enough translation quality-wise.